### PR TITLE
ETH-152: remove uniswap from package.json

### PR DESCRIPTION
**Pull request by jtakalai**
also added ethers to top-level for future scripts

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,5 +18,8 @@
     "graph2_copyInfo": "./scripts/copyAbis.sh",
     "graph3_build": "npm run doAll -w=streamregistry-thegraph-subgraph",
     "graph": "npm run graph1_build && npm run graph2_copyInfo && npm run graph3_build"
+  },
+  "dependencies": {
+    "ethers": "^5.0.0"
```
Comment (kare) [inline package.json:23]: @jtakalai Why was this added?

Comment (jtakalai) [reply package.json:23]: kind-of accidentally, but it will be used soon.
